QUEST_LV_0200.tsv

```diff
@@ -1063,32 +1063,32 @@ QUEST_LV_0200_20150317_001062	I am relieved for now.{nl}The rest will be okay wh
 QUEST_LV_0200_20150317_001063	Priest Raeli
 QUEST_LV_0200_20150317_001064	{memo X}Goddess Austeja is sealing two spots that are the source of the vicious energy.{nl}However, due to demons' continuous attacks, the seal is getting weaker.{nl}
 QUEST_LV_0200_20150317_001065	{memo X}That's why we lured the demons into demonic stones..{nl}Hasty behavior from the villagers created the trouble.
-QUEST_LV_0200_20150317_001066	You can never say that you are not responsible for this.
-QUEST_LV_0200_20150317_001067	I don't know if it is due to Medis Diena, but the vicious energy started to gather from deep inside the ground.{nl}The energy started to accumulate a lot and was about to explode.{nl}
-QUEST_LV_0200_20150317_001068	Goddess Austeja is trying her best to seal that energy.{nl}If she doesn't do that, this forest would turn into Vine Forest.{nl}
-QUEST_LV_0200_20150317_001069	The only living creatures are monsters.. I am talking about the land that much devastated.{nl}
-QUEST_LV_0200_20150317_001070	I want to explain about the circumstances a bit, but the monsters will come this way soon.{nl}To prepare for this kind of moment, we've prepared the protecting stone on the edge of the village.
-QUEST_LV_0200_20150317_001071	I am relieved by your words.{nl}If we don't protect this, the whole Siaulai forest may be destroyed so I am counting on you.{nl}
-QUEST_LV_0200_20150317_001072	I will explain it to you in details after you come back.{nl}Please get rid of the monsters.
+QUEST_LV_0200_20150317_001066	$There is no way you can say you have nothing to do with this.
+QUEST_LV_0200_20150317_001067	$I don't know if Medis Diena was the cause, but since some time ago, evil energy began to well deep underground.{nl}The energy has accumulated so much, it could explode anytime now.{nl}
```

Comment:
since some time ago, evil energy

That does not need a comma, and it causes the sentence to feel clunky.